src/ru/mangahub/src/eu/kanade/tachiyomi/extension/ru/mangahub/Mangahub.kt

```diff
@@ -83,16 +83,16 @@ open class Mangahub : ParsedHttpSource() {
         return GET("$baseUrl/explore/sort-is-update$pageStr", headers)
     }
 
-    override fun popularMangaSelector() = "div.comic-grid-col-xl"
+    override fun popularMangaSelector() = "div.item-grid"
 
     override fun latestUpdatesSelector() = popularMangaSelector()
 
     override fun popularMangaFromElement(element: Element): SManga {
-        val manga = SManga.create()
-        manga.thumbnail_url = element.select("div.fast-view-layer-scale").attr("data-background-image")
-        manga.title = element.select("a.fw-medium").text()
-        manga.setUrlWithoutDomain(element.select("a.fw-medium").attr("href"))
-        return manga
+        return SManga.create().apply {
+            thumbnail_url = element.selectFirst("img.item-grid-image")!!.attr("src")
```

Comment:
The URL set in `src` might be relative, use `.absUrl` instead of `.attr` to account for this.